src/main/webapp/i18n/de/programmingExercise.json

```diff
@@ -687,6 +687,17 @@
                 "pushExercise": "Lade den Aufgaben-Code in das entsprechende Repository. Es wird in <code>{{checkoutDirectory}}</code> ausgecheckt.",
                 "pushSolution": "Lade den Lösungs-Code in das entsprechende Repository. Es wird nicht ausgecheckt.",
                 "pushTest": "Lade den Test-Code in das entsprechende Repository. Es wird in <code>{{checkoutDirectory}}</code> ausgecheckt."
+            },
+            "dockerFlags": {
+                "disableNetworkAccess": {
+                    "title": "Netzwerkzugriff beim Ausführen der Studierendeneinreichung deaktivieren",
```

Comment:
Does this include local network or only to the internet? Maybe make this a bit clearer here or in the description